charts/pega/values.yaml

```diff
@@ -338,6 +338,8 @@ global:
         minAvailable: 1
         # maxUnavailable: "50%"
 
+configurations:
+  isCompressed: false
```

Comment:
Does it make sense to move this param under global? Also would this be a hidden param or we expose it to clients?

There is already a hidden param for mounting configurations in the global. I guess `.root.Values.global.configurations`. Should we consolidate?

Reply:
moved it under global and this will be exposed to CMC clients.